<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CanTcpPacket</class>
 <widget class="QDialog" name="CanTcpPacket">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>443</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CanSewsiPacket</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../AutosResourceFile.qrc">
    <normaloff>:/Images/Images/sample.ico</normaloff>:/Images/Images/sample.ico</iconset>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <widget class="QTabWidget" name="CanTcpPacketWidget">
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="CanTcpChoice">
      <attribute name="title">
       <string>Select Packet</string>
      </attribute>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>150</y>
         <width>431</width>
         <height>22</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Select Packet</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox">
          <item>
           <property name="text">
            <string>CAN</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>SEWSI</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="CanPacket">
      <attribute name="title">
       <string>CAN</string>
      </attribute>
      <widget class="QCheckBox" name="HexcheckBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>70</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Hex</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="DecimalcheckBox">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>30</y>
         <width>70</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Decimal</string>
       </property>
      </widget>
      <widget class="QLabel" name="CanIdLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>41</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>CAN ID</string>
       </property>
       <property name="textFormat">
        <enum>Qt::AutoText</enum>
       </property>
      </widget>
      <widget class="QLabel" name="DlcLabel">
       <property name="geometry">
        <rect>
         <x>15</x>
         <y>160</y>
         <width>41</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>DLC</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="CanIdlineEdit">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>80</y>
         <width>51</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="DlclineEdit">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>150</y>
         <width>51</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="Byte0lineEdit">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>230</y>
         <width>51</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="Byte1lineEdit">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>230</y>
         <width>51</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="Byte2lineEdit">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>230</y>
         <width>51</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="Byte3lineEdit">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>230</y>
         <width>51</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="Byte4lineEdit">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>230</y>
         <width>51</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="Byte5lineEdit">
       <property name="geometry">
        <rect>
         <x>440</x>
         <y>230</y>
         <width>51</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="Byte6lineEdit">
       <property name="geometry">
        <rect>
         <x>510</x>
         <y>230</y>
         <width>51</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="Byte7lineEdit">
       <property name="geometry">
        <rect>
         <x>580</x>
         <y>230</y>
         <width>51</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="MessageLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>230</y>
         <width>51</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>MESSAGE</string>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>437</x>
         <y>360</y>
         <width>201</width>
         <height>25</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QPushButton" name="CancelPushButton">
          <property name="text">
           <string>Cancel</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="OkPushButton">
          <property name="text">
           <string>Ok</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="TcpPacket">
      <attribute name="title">
       <string>TCP</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../AutosResourceFile.qrc"/>
 </resources>
 <connections/>
</ui>
